util/travis/common.sh

```diff
@@ -31,14 +31,27 @@ install_linux_deps() {
 # Mac OSX build only
 install_macosx_deps() {
 	brew update
-	brew install freetype gettext hiredis irrlicht jpeg leveldb libogg libvorbis luajit
+	brew install freetype gettext hiredis irrlicht 
+	brew upgrade jpeg 
+	brew install leveldb libogg libvorbis luajit
```

Comment:
merged install lines in one, kept install in below, new commit now available, still needs some testing.